source/install/config-proxy-nginx.rst

```diff
@@ -71,20 +71,28 @@ NGINX is configured using a file in the ``/etc/nginx/sites-available`` directory
 
 6. Restart NGINX.
 
-  On Ubuntu 14.04 and RHEL 6.6:
-  
-  ``sudo service nginx restart``
-  
-  On Ubuntu 16.04 and RHEL 7.1:
-  
-  ``sudo systemctl restart nginx``
+  On Ubuntu 14.04 and RHEL 6.6: ``sudo service nginx restart``
+
+  On Ubuntu 16.04 and RHEL 7.1: ``sudo systemctl restart nginx``
 
 7. Verify that you can see Mattermost through the proxy.
 
   ``curl http://localhost``
-  
+
   If everything is working, you will see the HTML for the Mattermost signup page.
 
-**What to do next**
+8. Make sure that Mattermost accepts connections only from the proxy.
```

Comment:
You'll still want to see the individual HA consoles to see if any are having trouble talking even if you cannot set props in the console.  You'll also need individual access to each console to call re-load config from disk.

Changing the bind address is a distant second to change an iptable or security group.  We should at the very least make a recommendation that it would be better to change this using iptables or security groups.  The bind feature really exists when you have multiple nic cards and want Mattermost to bind to only 1.